scripts/plugins/anchor-markdown-headings.js

```diff
@@ -42,8 +42,8 @@ module.exports = function anchorMarkdownHeadings (text, level, raw) {
 
   anchorTitle = anchorTitle.toLowerCase()
 
-  return '<h' + level + ' id="header-' + anchorTitle + '">' + text + '<a name="' +
-    anchorTitle + '" class="anchor" href="#' +
-    anchorTitle + '" aria-labelledby="header-' +
-    anchorTitle + '"></a></h' + level + '>'
+  const anchorId = `${slugger ? slugger.slug(anchorTitle) : anchorTitle}`
```

Comment:
Reading the [marked docs](https://github.com/markedjs/marked/blob/master/docs/USING_PRO.md) I get the impression that the `slugger` argument will _always_ be provided when our custom header renderer function (`anchorMarkdownHeadings()`) is called by marked. Or have I misunderstood?

If it is indeed always provided, could we invoke `slugger.slug()` directly, without having logic checking if `slugger` is indeed provided? Primarily to help new contributors and our future selfs when trying to understand what this code does later, not having to graps unnecessary logic is valuable.